src/webnn_native/openvino/ienn/build_ienn_unix.sh

```diff
@@ -72,10 +79,13 @@ cd "$build_dir"
 cmake -DCMAKE_BUILD_TYPE=Release "$CURRENT_PATH"
 make $NUM_THREADS
 
-# copy the libie_nn_c_api.so to ienn\lib\Linux64
-if [ -e "$CURRENT_PATH/lib/Linux64/libie_nn_c_api.so" ]; then
-    rm -rf "$CURRENT_PATH/lib/Linux64/libie_nn_c_api.so"
+# copy the libie_nn_c_api.so to webnn_native\out\Release
```

Comment:
Is the "webnn_native\out\Release" just an exmaple? Should we make this comment more generic?